web-client/src/modules/create/components/PostDetailsStep.tsx

```diff
@@ -0,0 +1,179 @@
+import { ArrowRightOutlined, CloseOutlined } from '@ant-design/icons';
+import { Form, Input, Select } from 'antd';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import TitleWithUnderline from 'src/components/TitleWithUnderline/TitleWithUnderline';
+import {
+  ButtonsDisplay,
+  DisplayButton,
+} from 'src/modules/create/components/DisplayElements';
+import { COLORS } from 'src/theme/colors';
+import styled from 'styled-components';
+
+const PostDetails: React.FC<PostDetailsProps> = ({
+  setPostDetails,
+  postDetails,
+  postTypes,
+  nextHandler,
+  prevHandler,
+}) => {
+  const { t } = useTranslation();
+  const [form] = Form.useForm();
+  const { Option } = Select;
+
+  const { title, type, body, customType } = postDetails;
+  const [showCustomType, setShowCustomType] = useState(type === 'customType');
+  const toggleCustomType = (value: string) => {
+    setShowCustomType(value === 'customType');
+  };
+
+  return (
+    <PostDetailsWrapper>
+      <TitleWithUnderline level={2} color={COLORS.primaryDark}>
+        {t('modules.create.postDetails.header')}
+      </TitleWithUnderline>
+      <DetailsForm
+        layout="vertical"
+        form={form}
+        onFinish={values => {
+          setPostDetails({ ...values, type: values.customType || values.type });
+          nextHandler();
+        }}
+        initialValues={{ title, type, body, customType }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+            alignContent: 'space-between',
+          }}
+        >
+          <Form.Item
+            label={t('modules.create.postDetails.titleLabel')}
+            name="title"
+            rules={[
+              {
+                required: true,
+                message: t('newRequest.form.title_error_message'),
+              },
+            ]}
+          >
+            <Input />
```

Comment:
In reference to my other comment, maybe replace this with: `<Input placeholder="[default/example value]"/>`

Reply:
fixed